src/components/TextInput/index.tsx

```diff
@@ -1,70 +1,59 @@
 /** @jsx jsx */
 import * as React from "react";
 import { jsx } from "@emotion/core";
-import Text from "../Text";
+import styled from "lib/styled";
+import { spaceSet, SpaceSetProps, fontSize, FontSizeProps } from "onno-react";
+import Text from "components/Text";
+import Box from "components/Box";
 
-type FontSizes = 1 | 2 | 3 | 4 | 5;
+type TextInputProps = SpaceSetProps & FontSizeProps;
 
-interface Props {
-  label?: string;
+interface Props extends TextInputProps {
+  placeholder?: string;
 }
 
-const TextInput: React.FC<Props> = ({ label = "Label", ...props }) => {
-  const [focused, setFocus] = React.useState<boolean>(false);
-  const [filled, setFilled] = React.useState<boolean>(false);
-  const [labelSize, setLabelSize] = React.useState<FontSizes>(5);
+const TextInput = styled.input<Props>(spaceSet, fontSize, props => ({
+  border: "none",
+  borderBottom: `1px solid ${props.theme.colors.dark}30`,
+  width: "100%",
+}));
+
+export const TextField: React.FC<Props> = props => {
+  const [isFocus, setIsFocus] = React.useState(false);
```

Comment:
What about `isFocused` and `setIsFocused`. Or simply `focus` and `setFocus`? Currently it doesn't read well.